config/mcp_config.py

```
@@ -0,0 +1,81 @@
+                "timeout": 30,
+            },
+        }
+
+    def get_config(self) -> Dict[str, Any]:
+        """Get current configuration"""
+        if self.config_path.exists():
+            try:
+                with open(self.config_path, "r") as f:
+                    return json.load(f)
+            except Exception:
+                pass
```

Ruff (0.11.9)

`try`-`except`-`pass` detected, consider logging the exception

(S110)